cmd/kubeadm/app/master/manifests_test.go

```diff
@@ -567,9 +539,9 @@ func TestGetAPIServerCommand(t *testing.T) {
 				"--secure-port=123",
 				"--allow-privileged=true",
 				"--kubelet-preferred-address-types=InternalIP,ExternalIP,Hostname",
+				"--experimental-bootstrap-token-auth=true",
```

Comment:
I'm going to let this go but there is a bunch of stuff in here that is just drive by clean up.  Good stuff but creates noise in the PRs.